tox.ini

```diff
@@ -66,11 +66,11 @@ commands =
    pre-commit run [] --all-files --show-diff-on-failure
 
 [testenv:vendoring]
-basepython = python3.8
+basepython = python3
 skip_install = True
 commands_pre =
 deps =
-    vendoring==0.2.2
+    vendoring>=0.3.3
```

Comment:
```suggestion
    vendoring~=0.3.3
```

Reply:
I don't understand why we should pin to the minor version of a package that is *especially* made for pip, but since you maintain both, I'm happy to go along with it :smile: